### PR TITLE
Adjust content on the school claim screen to help users who worked in multiple schools

### PR DESCRIPTION
**Pull request by pezholio**
This adds the copy added by Faye to the school search page and the relevant ineligibility pages. Here's how they look:

image

image

image

### CODE REVIEW

**Comment by pezholio** [comment]
[Adjust content on the school claim screen to help users who worked in multiple schools](https://trello.com/c/jdijvr6l/902-adjust-content-on-the-school-claim-screen-to-help-users-who-worked-in-multiple-schools)

File: app/views/claims/ineligible.html.erb [hunk 1]
```diff
@@ -2,14 +2,12 @@
...
-      You’re not eligible for this payment
-    </h1>
-
     <%= render partial: "ineligibility_reason_#{current_claim.eligibility.ineligibility_reason}", locals: { claim_school_name: current_claim.eligibility.claim_school_name, current_school_name: current_claim.eligibility.current_school_name } %>
 
     <p class="govuk-body">
-      You can find more information on the <%= link_to "guidance for this service", tslr_guidance_url, class: 'govuk-link' %>.
+      For more information about the eligibility criteria, or if you need help with
+      your claim, contact
+      <%= mail_to "studentloanteacherpayment@digital.education.gov.uk", "studentloanteacherpayment@digital.education.gov.uk", class: "govuk-link" %>.
```
Comment (robbiepaul) [inline app/views/claims/ineligible.html.erb]: We could use the `support_email_address(params[:policy])` helper here
```suggestion
      <%= mail_to support_email_address(params[:policy]), support_email_address(params[:policy]), class: "govuk-link" %>.
```

File: app/views/claims/_ineligibility_reason_not_taught_eligible_subjects.html.erb [hunk 2]
```diff
@@ -10,3 +18,12 @@
   <li>computer science</li>
   <li>languages (not including English)</li>
 </ul>
+
+<p class="govuk-body">
+  If you taught at more than one school between April 6 2018 and April 5 2019,
+  you can <%= link_to "start your claim again", new_claim_path %> with another
```
Comment (robbiepaul) [inline app/views/claims/_ineligibility_reason_not_taught_eligible_subjects.html.erb]:
```suggestion
  you can <%= link_to "start your claim again", new_claim_path, class: "govuk-link" %> with another
```

File: app/views/claims/_ineligibility_reason_ineligible_claim_school.html.erb [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
-  April 2019, is not an eligible school. You can only get this payment if you
-  were employed at an eligible school.
+  <%= claim_school_name %> is not an eligible school. You can only get this
+  payment if you  were employed at an eligible school between 6 April 2018
+  and 5 April 2019.
+</p>
+
+<p class="govuk-body">
+  If you taught at more than one school during this period, you can
+  <%= link_to "start your claim again", new_claim_path %> with another school.
```
Comment (robbiepaul) [inline app/views/claims/_ineligibility_reason_ineligible_claim_school.html.erb]:
```suggestion
  <%= link_to "start your claim again", new_claim_path, class: "govuk-link" %> with another school.
```

**Pull request merged**